fs-v1-metadata.go

```diff
@@ -78,6 +79,33 @@ func newFSMetaV1() (fsMeta fsMetaV1) {
 	return fsMeta
 }
 
+// newFSFormatV1 - initializes new formatConfigV1 with FS format info.
+func newFSFormatV1() (fsFormatMeta formatConfigV1) {
+	fsFormatMeta = formatConfigV1{}
+	fsFormatMeta.Version = "1"
+	fsFormatMeta.Format = "fs"
+	fsFormatMeta.FS = &fsFormat{}
+	fsFormatMeta.FS.Version = "1"
+	return fsFormatMeta
+}
+
+// writes FS format (format.json) into minioMetaBucket.
+func writeFSFormatData(storage StorageAPI, fsFormat formatConfigV1) error {
+	metadataBytes, err := json.Marshal(fsFormat)
+	if err != nil {
+		return err
+	}
+	// fsFormatJSONFile - format.json file stored in minioMetaBucket(.minio) directory.
+	n, err := storage.AppendFile(minioMetaBucket, fsFormatJSONFile, metadataBytes)
```

Comment:
what happens when the file already exists? this will corrupt the previous `format.json`